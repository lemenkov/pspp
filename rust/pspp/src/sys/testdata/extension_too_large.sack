# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; 2; 1; 0; -1; 100.0; "01 Jan 11"; "20:53:52"; s64 ""; i8 0 *3;

# Numeric variable, no label or missing values.
2; 0; 0; 0; 0x050800 *2; s8 "NUM1";

# Too-large extension record.
7; 3; >>0xfffff000 * 2<<;
