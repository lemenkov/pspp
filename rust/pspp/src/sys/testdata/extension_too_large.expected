At offset 0xd8, record type 7 subtype 3 is too large with element size 4294963200 and 4294963200 elements.
